dev/archery/archery/benchmark/google.py

```diff
@@ -87,9 +89,18 @@ class GoogleBenchmarkObservation:
     RegressionSumKernel/32768/0_stddev          0 us          0 us  288.046MB/s
     """
 
-    def __init__(self, name, real_time, cpu_time, time_unit, run_type,
-                 size=None, bytes_per_second=None, items_per_second=None,
-                 **counters):
+    def __init__(
+        self,
+        name,
+        real_time,
+        cpu_time,
+        time_unit,
+        run_type,
+        size=None,
+        bytes_per_second=None,
+        items_per_second=None,
+        **counters,
+    ):
```

Comment:
This is the kind of formatting that I'd like to avoid. Can it be configured?

Reply:
Good question, taking a look at it.